packages/plugins/other/visitor-plugin-common/src/base-resolvers-visitor.ts

```diff
@@ -675,8 +674,8 @@ export class BaseResolversVisitor<
   protected _hasReferencedResolversUnionTypes = false;
   protected _hasReferencedResolversInterfaceTypes = false;
   protected _resolversUnionTypes: Record<string, string> = {};
-  protected _resolversUnionParentTypes: Record<string, string> = {};
```

Comment:
`_resolversUnionParentTypes` was used previously but not needed anymore. Removing in this PR as a drive-by